internal/core/config.go

```diff
@@ -19,10 +19,29 @@
 package core
 
 import (
+	"regexp"
+
 	"github.com/sapcc/castellum/internal/db"
 )
 
 // Config contains the app-level configuration options.
 type Config struct {
-	MaxAssetSize map[db.AssetType]*uint64
+	MaxAssetSize []MaxAssetSizeRule
```

Comment:
The naming is not really descriptive of what's in this field:
```suggestion
	MaxAssetSizeRules []MaxAssetSizeRule
```